src/Nethermind/Nethermind.JsonRpc/Modules/Eth/EthModule.cs

```diff
@@ -357,101 +358,17 @@ public async Task<ResultWrapper<Keccak>> eth_sendRawTransaction(byte[] transacti
             }
         }
 
-        public ResultWrapper<string> eth_call(TransactionForRpc transactionCall, BlockParameter blockParameter = null)
-        {
-            SearchResult<BlockHeader> searchResult = _blockFinder.SearchForHeader(blockParameter);
-            if (searchResult.IsError)
-            {
-                return ResultWrapper<string>.Fail(searchResult);
-            }
-
-            BlockHeader header = searchResult.Object;
-            if (!HasStateForBlock(header))
-            {
-                return ResultWrapper<string>.Fail($"No state available for block {header.Hash}",
-                    ErrorCodes.ResourceUnavailable);
-            }
-
-            FixCallTx(transactionCall);
-
-            using CancellationTokenSource cancellationTokenSource = new(_cancellationTokenTimeout);
-            CancellationToken cancellationToken = cancellationTokenSource.Token;
-            Transaction tx = transactionCall.ToTransaction(_blockchainBridge.GetChainId());
-            BlockchainBridge.CallOutput result = _blockchainBridge.Call(header, tx, cancellationToken);
+        public ResultWrapper<string> eth_call(TransactionForRpc transactionCall, BlockParameter? blockParameter = null) =>
```

Comment:
do not use => for non simple returns

Reply:
Its simple return in my book, object creation is only for virtual call here